functions-and-operators/string-functions.md

```diff
@@ -96,3 +96,19 @@ The value options of `match_type` between TiDB and MySQL are:
     For example, the `SELECT REGEXP_LIKE(a, b, "n") FROM t1` in MySQL is the same as the `SELECT REGEXP_LIKE(a, b, "s") FROM t1` in TiDB.
 
 - TiDB does not support `"u"`, which means Unix-only line endings in MySQL.
+
+### Data type compatibility
+
+The difference between TiDB and MySQL in binary string type:
+
+- Since MySQL 8.0.22, regular expression functions do not support binary strings. For more details, refer to [MySQL documentation](https://dev.mysql.com/doc/refman/8.0/en/regexp.html). But in practice, regular functions could work when all parameters or return types are binary strings. Otherwise, an error will be reported.
```

Comment:
```suggestion
- MySQL deos not support binary strings in regular expression functions since 8.0.22. For more details, refer to [MySQL documentation](https://dev.mysql.com/doc/refman/8.0/en/regexp.html). But in practice, regular functions can work in MySQL when all parameters or return types are binary strings. Otherwise, an error will be reported.
```